### ISSUE TITLE
[conda napari] pip used by default

### ISSUE THREAD

**Issue by psobolewskiPhD** [napari/napari-plugin-manager#73]
I assumed that with conda-forge napari I'd get plugins from conda-forge by default, but no:
```
napari                    0.5.0                hd8ed1ab_2    conda-forge
napari-base               0.5.0              pyh9208f05_2    conda-forge
napari-console            0.0.9              pyh9208f05_0    conda-forge
napari-ome-zarr           0.6.1                    pypi_0    pypi
napari-plugin-engine      0.2.0              pyhd8ed1ab_2    conda-forge
napari-plugin-manager     0.1.0a2            pyhd8ed1ab_0    conda-forge
napari-svg                0.2.0              pyhd8ed1ab_0    conda-forge
```
did a conda-forge install of napari and then used the manager to install `napari-ome-zarr`

**Comment by jaimergp** [comment]
xref #46 - can we get some feedback there?

**Comment by jaimergp** [comment]
Oh, this was addressed in #47. nvm
